core/foundation/inc/RStringView.h

```diff
@@ -29,7 +29,11 @@
 namespace std {
 
    template<class _CharT, class _Traits = std::char_traits<_CharT> >
+// What happens if we don't have std::basic_string_view and
+// std::experimental::basic_string_view?
```

Comment:
Then ROOT injects its own `string_view` into `std::`. Could you remove the comment?